Library/Homebrew/cmd/update.sh

```diff
@@ -9,6 +9,14 @@
 # shellcheck source=/dev/null
 source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
 
+git() {
```

Comment:
I think it would be worth mentioning (in a comment) that this replaces the implementation in `Library/brew.sh` and why it does that.